File: pset6/mario/mario.py
height = 0
while True:
    try:
        height = int(input("Height: "))
        if (height >= 1) and (height <= 8):
            break
    except:
        pass
i = 1
while(i <= height):
    print(" " * (height - i), end="")
    print("#" * i, end="")
    print("  ", end="")
    print("#" * i, end="")
    print()
    i = i + 1

File: pset1/mario/mario.c
#include<stdio.h>
#include<cs50.h>
int main(void)
{
    int n = 0, i = 0, j = 0, k = 0;
    do
    {
        n = get_int("Height:\n");
    }
    while (!(n >= 1 && n <= 8));
    for (i = 0; i < n; i++)
    {
        for (j = i; j < n - 1; j++)
        {
            printf(" ");
        }
        for (k = 0; k <= i; k++)
        {
            printf("#");
        }
        printf("  ");
        for (k = 0; k <= i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
    return 0;
}
